packages/core/src/app/address/AddressFormModal.tsx

```diff
@@ -59,13 +58,12 @@ const SaveAddress: FunctionComponent<
                 shouldShowSaveAddress={false}
             />
             <div className="form-actions">
-                <a
-                    className="button optimizedCheckout-buttonSecondary"
-                    href="#"
-                    onClick={preventDefault(onRequestClose)}
-                >
+                <Button
+                    onClick={onRequestClose}
```

Comment:
This seems like a change in the overall behaviour of the modal in general? Is that expected?

Reply:
The PR looks good to me besides this question.
I did notice we have three `cancel` buttons using the `<Button /`> not `<a/>`.

Reply:
Might be good to go in the commit history to see if there is a reason 🤔

Reply:
Looking into history, this has not been changed since it was first implemented ([ref](https://github.com/bigcommerce/checkout-js/commit/450c9f87c241e59e8a72ae26dc407fe1f5b08a4e#diff-7223c63a2de6c7af5e87e73bc696db9509af45606519268b5f58a0cf62ab28d4)). I guess it is implemented as `<a>` tag because we do not have button style to look like a link, no other reason that I can think of. Even in this PR, I have added [this style](https://github.com/bigcommerce/checkout-js/pull/2107/files#diff-3db8c7765aebecc0f9859a3c3f636d8227097b2c550bd3c98a01ce101bbe2a2e) to remove button border to make it look like an actionable text. We can change it back to `<a>` tag if you suggest. Moreover, this modal is only used in multi shipping (old and new both)

Reply:
Thanks @richapatel1510 for the detailed analysis, looks good to me to change to Button in this case. We just have to be diligent around the changes.